src/sentry/static/sentry/app/views/performance/charts/styles.tsx

```diff
@@ -15,6 +15,12 @@ export const SectionHeading = styled('h4')`
   line-height: 1.2;
 `;
 
+export const SectionValue = styled('span')`
```

Comment:
We have another copy of this in eventsV2/styles. Instead of duplicating could we consolidate the two copies into `components/charts`?